SUSE/rmt
# Release 2.16

Release 2.16 and remove unwanted `.old_repodata` feature which use case is unknown.

part of: https://trello.com/c/hjXMzOgz/3275-rdeb-release-rmt-216

**How to test this pull request:**

1) Check the documentation changes if they make sense
2) Run development RMT with this branch applied

```
# Check debian repository mirroring works
$ bin/rmt-cli repos custom add http://ppa.launchpadcontent.net/bhdouglass/clickable/ubuntu/dists/lunar/ clickable
$ bin/rmt-cli mirror repository clickable
$ tree public/repo/bhdouglass/clickable/ubuntu/
# expect: tree structure includes metadata as expected

# Check repomd repository mirroring works and no .old_repodata is created
$ bin/rmt-cli repos custom add https://rpms.remirepo.net/enterprise/8/php80/x86_64/ php80
$ bin/rmt-cli mirror repository php80
$ tree public/repo/enterprise/8/php80/ 
# expect: tree structure looks okay
$ ls -la public/repo/enterprise/8/php80/ 
# expect: no .old_repodata exists
```
**As always, if you have question regarding this pull request, please do not hesitate to reach out to me!**:rocket:

Thank you very much!
